Mihai-Blaga/COMP30022
# Feature/AA-59 google authentication

## Test

- Logout of google account from emulator
- Open the CRM app, it should prompt for a login
- After logging in once, the account details will be stored
- Close and reopen the app after logging in, it should skip the login screen since it automatically logins for you
